node/node.go

```diff
@@ -57,7 +57,7 @@ import (
 	"github.com/filecoin-project/go-filecoin/porcelain"
 	"github.com/filecoin-project/go-filecoin/proofs"
 	"github.com/filecoin-project/go-filecoin/proofs/sectorbuilder"
-	bapi "github.com/filecoin-project/go-filecoin/protocol/block"
+	"github.com/filecoin-project/go-filecoin/protocol/block"
```

Comment:
Please keep this as `bapi` here. Block is so overloaded and in the context of the node package readers will assume it means many other things.